src/main/java/appeng/worldgen/MeteoritePlacer.java

```diff
@@ -91,7 +91,18 @@ public MeteoritePlacer()
 		this.validSpawn.add( Blocks.ice );
 		this.validSpawn.add( Blocks.snow );
 		this.validSpawn.add( Blocks.stained_hardened_clay );
+		
+		for( final String name : AEConfig.instance.meteoriteValidSpawnBlocks )
+		{
+			Block block = Block.getBlockFromName( name );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'block' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)